pages/gm/w22.js

```diff
@@ -0,0 +1,219 @@
+import { NextSeo } from 'next-seo';
+import Link from 'next/link';
+import React from 'react';
+import Countdown from 'react-countdown';
+
+import Banner from '../../components/Banner';
+import Layout from '../../components/Layout';
+
+const GM_START_TIME = new Date('2022-01-06T18:30-07:00');
+const RSVP_LINK = 'http://links.uclaacm.com/wintergm-rsvp-22';
+const SLIDES_LINK = 'https://docs.google.com/presentation/d/1XGyJdmuR78C9liEPlwqMq2PpNSaXS-PtlK15fvzQLRE/edit?usp=sharing';
+// const ZOOM_LINK = '#';
```

Comment:
Assuming this will be added in followupPR

Reply:
Yep!